samples/angular-todo/src/app/features/journey/form/form.component.html

```diff
@@ -154,6 +154,27 @@ <h1 class="text-center fs-2 mb-3" *ngIf="title">{{ title }}</h1>
                 (setAnswer)="$any(callback).setAnswer($event)"
               ></app-kba>
 
+              <!--
+                Handle a KbaCreateCallback using the Kba component.
+                Emitted questions and answers are used to set the values on the callback.
+              -->
+              <app-identity-provider
+                *ngSwitchCase="'SelectIdPCallback'"
+                [callback]="$any(callback)"
+                [name]="callback?.payload?.input?.[0]?.name"
```

Comment:
getInput?

Reply:
it's the same for the other components. Followed the standard